packages/mrm-task-gitignore/index.js

```diff
@@ -14,13 +15,15 @@ const ignores = [
 	'*.sublime-workspace',
 	'*.log',
 	'.eslintcache',
+	existsSync('package-lock.json') ? 'yarn.lock' : '',
```

Comment:
`existsSync('package-lock.json') && 'yarn.lock'` would be more clear probably.

Reply:
That would change the type of ignores to `(string | bool)[]`, I preferred to keep it `string[]`

Reply:
Using empty sting instead of an undefined or false is a hack anyway.